src/pages/coa/components/COATableTree.vue

```diff
@@ -121,11 +121,18 @@ export default {
       this.loadAccounts()
       this.addColumnOnTable()
       this.addStyleForExchangeColumn()
+    },
+    tokensWithUserSort () {
+      this.columns = this.columns.slice(0, 2)
+      this.setUpColumns()
+      this.addColumnOnTable()
+      this.addStyleForExchangeColumn()
     }
   },
   computed: {
     ...mapState('contAccount', ['treeAccounts']),
     ...mapState('tokens', ['tokensWithUserExange']),
+    ...mapState('tokens', ['tokensWithUserSort']),
```

Comment:
we can add tokensWitjUserSort in the same line that tokensWithUserExange like an other object